service.go

```diff
@@ -60,6 +71,49 @@ func getTenantInfo() (string, int64, bool) {
 	return AssignTenantID.Load().(string), StartTime.Load(), IsUnassigning.Load()
 }
 
+func GetTiFlashTaskNum() (int, error) {
+	client := http.Client{
+		Timeout: 5 * time.Second,
+	}
+	resp, err := client.Get(TiFlashMetricURL)
+	if err != nil {
+		return 0, err
+	}
+	if resp.StatusCode != 200 {
+		return 0, errors.New("http status code is not 200")
+	}
+	defer resp.Body.Close()
+	data, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return 0, err
+	}
+	res, err := GetTiFlashTaskNumByMetricsByte(data)
+	if err != nil {
+		return 0, err
+	}
+	return res, nil
+}
+
+func GetTiFlashTaskNumByMetricsByte(data []byte) (int, error) {
+	res := 0
+	reader := bytes.NewReader(data)
+	var parser expfmt.TextParser
+	metricFamilies, err := parser.TextToMetricFamilies(reader)
+	if err != nil {
+		return 0, err
+	}
+	for _, v := range metricFamilies {
+		if strings.HasPrefix(*v.Name, "tiflash_coprocessor_handling_request_count") {
```

Comment:
replace this magic number： `tiflash_coprocessor_handling_request_count` with var declartion